src/sentry/static/sentry/app/views/onboarding/projectSetup/projectDocs.jsx

```diff
@@ -249,6 +285,13 @@ const DocsWrapper = styled(posed.div(docsTransition))`
     margin-bottom: 1rem;
   }
 
+  /**
+   * XXX(epurkhiser): This comes from the doc styles and avoids bottom margin issues in alerts
+   */
+  .content-flush-bottom *:last-child {
```

Comment:
Can we style the alert instead of relying on `last-child`?

Reply:
No these classes come from the html generated by `docs.sentry.io`'s API.

Reply:
ah actually this should be in a different PR